motion_control/ArmGroup joints
---
geometry_msgs/Pose eePose
